webpack.example-app.config.js

```diff
@@ -14,12 +14,12 @@ prodConfig.output = {
   filename: 'example.js',
 };
 
-prodConfig.module.rules.push(
-  {
-    test: /\.css$/,
-    loader: 'style-loader!css-loader?sourceMap',
-  },
-);
+// prodConfig.module.rules.push(
+//   {
+//     test: /\.css$/,
+//     loader: 'style-loader!css-loader?sourceMap',
+//   },
+// );
```

Comment:
Why did you have to comment this?
Is it meant to stay?

It's true that we could get rid of css modules completely for the examples, doesn't bring much anyway.

Reply:
I do this so I can debug the example code. It could probably be easier but it's not that difficult to change a couple lines of code.